code/datums/components/butchering.dm

```diff
@@ -43,7 +43,7 @@
 			new sinew (T)
 		meat.guaranteed_butcher_results.Remove(sinew)
 	if(butcher)
-		meat.visible_message("<span class='notice'>[butcher] butchers [meat].</span>")
+		meat.visible_message("<span class='none'>[butcher] butchers [meat].</span>")
```

Comment:
If you're keeping unstyled classes, the class name probably shouldn't be none. In CSS this tends to mean it doesn't show.

Reply:
You mean specifically `span class='none'`? Could it be `span class='plain'` or something similar?

Reply:
Yeah plain is fine, just the word none tends to imply a class that has thever "display: none" property, which hides elements.